src/Components/SmartComponents/Modals/CvePairStatusModal.js

```diff
@@ -121,14 +121,16 @@ export const CvePairStatusModal = ({ cves, updateRef, inventories, hasDifferentS
     return (
         <BaseModal items={cveList} onSave={handleSave} onSuccessNotification={successNotification} title={modalTitle}>
             <Stack gutter={'md'}>
-                <StackItem>
-                    {hasDifferentStatus ?
+                {hasDifferentStatus ?
```

Comment:
you don't have to use Ternary Operator. You can simply  use  `&&`
`{ myVariable && <MyComponent /> } `

Reply:
@katerinapat This change caused https://projects.engineering.redhat.com/browse/VMAAS-1039 (type error) so I am going to make a PR, where I revert it.